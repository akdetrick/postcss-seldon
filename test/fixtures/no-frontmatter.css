/**doc
Just some good old markdown here
*/
