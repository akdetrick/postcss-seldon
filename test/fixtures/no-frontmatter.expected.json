[
	{
		"meta": null,
    "markdown": "Just some good old markdown here",
    "html": "<p>Just some good old markdown here</p>\n"
	}
]
